plugins/woocommerce/src/Admin/API/Reports/Orders/DataStore.php

```diff
@@ -398,11 +403,14 @@ protected function include_extended_info( &$orders_data, $query_args ) {
 				'products' => array(),
 				'coupons'  => array(),
 				'customer' => array(),
+				'origin'   => '',
```

Comment:
🚧 The `attribution` property is an `object`. Hence the default should be an empty `array` instead of a `string`.
```suggestion
				'products'    => array(),
				'coupons'     => array(),
				'customer'    => array(),
				'attribution' => array(),
```